### PR TITLE
Add log-only metric reporter.

### PR DESCRIPTION
**Pull request by michelle192837**

### CODE REVIEW

**Comment by fejta** [comment]
I think you need to rerun update-bazel and/or `git add -A`

**Review by fejta — COMMENTED** [review]
Would I be annoying to ask that we use it somewhere? 

For example: https://github.com/GoogleCloudPlatform/testgrid/blob/14ec4b9a9e7aa10f59072d783a9b9d37bb05b4dd/cmd/updater/main.go#L137-L138

**Comment by michelle192837** [comment]
> Would I be annoying to ask that we use it somewhere?
> 
> For example:
> 
> https://github.com/GoogleCloudPlatform/testgrid/blob/14ec4b9a9e7aa10f59072d783a9b9d37bb05b4dd/cmd/updater/main.go#L137-L143

Np! Does this work?

File: cmd/updater/main.go [hunk 1]
```diff
@@ -136,9 +137,11 @@ func main() {
 	groupUpdater := updater.GCS(opt.groupTimeout, opt.buildTimeout, opt.buildConcurrency, opt.confirm, updater.SortStarted)
 	updateOnce := func() {
 		start := time.Now()
+		cycle := metrics.NewLogInt64("cycle_duration", "How long an update cycle took, in seconds.", logrus.New())
```
Comment (fejta) [inline cmd/updater/main.go:140]: Shouldn't we just create this once, before updateOnce definition? (so L137 or so?)

Comment (michelle192837) [reply cmd/updater/main.go]: Yup, thanks for catching! 

Also manually verified that this will successfully log when the Updater is run.

**Review by fejta — APPROVED** [review]

**Pull request merged**